dev_guide/builds.adoc

```diff
@@ -17,6 +17,74 @@ runnable images to be used on OpenShift. There are three types of builds:
 - STI build
 - custom build
 
+== Defining a buildConfig
+
+A `*buildConfig*` describes a single link:templates.html[template] and a
+set of link:#triggers[triggers] for when a new build should be created.
+
+A buildConfig is a REST object, which can be used in a POST to the API server to
+create a new instance. The following configuration results in a new build
+every time a Docker image tag or the source code changes:
+
+[source.json]
+----
+{
+  "kind": "BuildConfig",
+    "apiVersion": "v1beta3",
+    "metadata": {
+      "name": "ruby-sample-build <1>",
+    },
+  "spec": {
+    "triggers": [ <2>
+      {
+        "type": "github",
+        "github": {
+          "secret": "secret101"
+        }
+      },
+      {
+        "type": "generic",
+        "generic": {
+          "secret": "secret101"
+        }
+      },
+      {
+        "type": "imageChange",
+        "imageChange": {}
```

Comment:
Seems that the whole ImageChange trigger has been removed, that's a mistake.
What @bparees meant is that `"imageChange": {}` is not needed in API v1.